packages/notifications/migrations/sqls/20180628154012-devices-up.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE "devices" (
+  "id"            uuid primary key not null default uuid_generate_v4(),
+  "userId"        uuid not null references users(id),
+  "sessionId"     uuid not null unique references sessions(id) ON UPDATE CASCADE ON DELETE CASCADE,
+  "token"         text unique,
+  "information"   jsonb,
```

Comment:
I would have liked `info` the most, as it's device information. The only one here really. But because the App would need to adapt, I think we should let it as it is.